iconservice/deploy/icon_score_deploy_engine.py

```diff
@@ -145,7 +146,7 @@ def _score_deploy(self, context: 'IconScoreContext', tx_params: 'IconScoreDeploy
         content_type = data.get('contentType')
 
         if content_type == 'application/tbears':
-            if not context.legacy_tbears_mode:
+            if not IconScoreContextUtil.legacy_tbears_mode:
```

Comment:
I can not find the reason why "legacy_tbears_mode" should be moved from IconScoreContext to IconScoreContextUtil.

Reply:
I understand.
rollback this